markup/goldmark/integration_test.go

```diff
@@ -711,3 +711,76 @@ echo "hello";
 	b := hugolib.Test(t, files)
 	b.AssertFileContent("public/index.html", "<div class=foo><?php\necho \"hello\";\n?>\n</div>")
 }
+
+// Issue #10894
+func TestLatexInlineFences(t *testing.T) {
+	t.Parallel()
+
+	files := `
+-- config.toml --
+[markup.goldmark.extensions.passthrough]
+enable = true
+inlineDelimiters = [
+  {open = "$", close = "$"},
+]
+-- content/p1.md --
+---
+title: "p1"
+---
+## LaTeX test
+
+Inline equation that would be mangled by default parser: $a^*=x-b^*$
+
+-- layouts/_default/single.html --
+{{ .Content }}
+`
+
+	b := hugolib.NewIntegrationTestBuilder(
```

Comment:
There is a convience func that makes these tests shorter:

```go
b := hugolib.Test(t, files)
b.Assert...
```

Reply:
Done